examples/resource-management/02-Resource.sh

```diff
@@ -3,44 +3,44 @@ set -e
 printf "\n=== Managing Resources in Azure ===\n"
 
 printf "\n1. Creating a new resource group: %s and location: %s.\n" "$groupName" "$location"
-az group create --name "$groupName" --location "$location"
+az resourcemanager group create --name "$groupName" --location "$location"
 destinationGroupName=$groupName"Destination"
 
 printf "\n2. Registering Resource Provider Namespace.\n"
 providerNamespace="Providers.Test"
-az resource provider register --ProviderNamespace $providerNamespace --Force
+az resourcemanager resource resourcemanager provider register --ProviderNamespace $providerNamespace --Force
```

Comment:
@BurtBiel so this is clearly incorrect - even if we keep resourceManager, it should not appear twice.